lib/presentation/redux/states/my_space_state.dart

```diff
@@ -39,30 +41,55 @@ import 'package:linshare_flutter_app/presentation/redux/states/linshare_state.da
 
 @immutable
 class MySpaceState extends LinShareState {
-  final List<Document> documentList;
+  final Map<Document, bool> documentList;
```

Comment:
About data structure concept, I think we use `Map` here is not appropriate. We use `List` when we need listing something, special for the things need to be in order, and `Map` for input and output. 

I think `Document` list may sometimes need to be ordered by date, name,... And I think we use `List` is more appropriate than `Map`

Reply:
I think we can consider to create a wrapper class included two properties is document, and select status. We also need to consider abstract it. It could be:
```
class SelectableElement<T> {
    T element;
    SelectMode selectMode;
}
```
We can reuse it with any type of element

Reply:
Love ur idea, check again please :)